src/main/java/net/sf/jabref/model/entry/FileField.java

```diff
@@ -73,17 +62,21 @@ public ParsedFileField(String description, String link, String fileType) {
 
         @Override
         public boolean equals(Object o) {
-            if (this == o)
+            if (this == o) {
                 return true;
-            if (o == null || this.getClass() != o.getClass())
+            }
+            if ((o == null) || (this.getClass() != o.getClass())) {
                 return false;
+            }
 
             FileField.ParsedFileField that = (FileField.ParsedFileField) o;
 
-            if (this.description != null ? !this.description.equals(that.description) : that.description != null)
+            if (this.description != null ? !this.description.equals(that.description) : that.description != null) {
                 return false;
-            if (this.link != null ? !this.link.equals(that.link) : that.link != null)
+            }
+            if (this.link != null ? !this.link.equals(that.link) : that.link != null) {
```

Comment:
Hm, I do not like a ternary construct within an if condition, as it makes this code extremely hard to read. Could you extract the result of the ternary expression into a separate variable, please?